docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md

```diff
@@ -6,142 +6,31 @@ toc_min_heading_level: 2
 toc_max_heading_level: 2
 ---
 
-import TabItem from '@theme/TabItem';
-import Tabs from '@theme/Tabs';
+A Data Asset is a collection of records from a Data Source. You can validate the whole Data Asset or a time-based subset of it. When you first connect to a Data Source, you define a minimum of one Data Asset. You can add more Data Assets from that same Data Source later. 
 
-A Data Asset is a collection of records that you create when you connect to your Data Source. When you connect to your Data Source, you define a minimum of one Data Asset. You use these Data Assets to create the Batch Requests that select the data that is provided to your Expectations.
+## Add a Data Asset from an existing Data Source
 
-<!-- [//]: # (TODO: To learn more about Data Assets, see Data Asset.) -->
-
-
-## Create a Data Asset
-
-Create a Data Asset to define the data you want GX Cloud to access. To connect to Data Assets for a Data Source not currently available in GX Cloud, see [Connect to data](/core/connect_to_data/connect_to_data.md) in the GX Core documentation. 
-
-<Tabs
-  groupId="manage-data-assets"
-  defaultValue='Snowflake'
-  values={[
-  {label: 'Snowflake', value:'Snowflake'},
-  {label: 'PostgreSQL', value:'PostgreSQL'},
-  ]}>
-<TabItem value="Snowflake">
-
-Define the data you want GX Cloud to access within Snowflake. 
+Define the data you want GX Cloud to access. 
 
 ### Prerequisites
 
-- You have a [GX Cloud account](https://greatexpectations.io/cloud).
-
-- You have a Snowflake database, schema, and table.
+- You have connected GX Cloud to the relevant Data Source.
 
-- You have a [Snowflake account](https://docs.snowflake.com/en/user-guide-admin) with USAGE privileges on the table, database, and schema you are validating, and you have SELECT privileges on the table you are validating. To improve data security, GX recommends using a separate Snowflake user service account to connect to GX Cloud.
+:::tip New Data Source?
+To add a Data Asset from a new Data Source, refer to [Connect GX Cloud](/cloud/connect/connect_lp.md).
+:::
 
-- You know your Snowflake password.
-
-### Connect to a Snowflake Data Asset
+### Procedure
 
 1. In GX Cloud, click **Data Assets** > **New Data Asset**.
```

Comment:
General question for all the sections: Do you think we need to specify where **Data Assets** is in the UI for that initial click?